sources/packages/web/src/components/partial-view/DesignationAgreement/DesignationAgreementForm.vue

```diff
@@ -35,7 +35,7 @@ export default {
     const MANAGE_LOCATIONS_LINK = "goToManageLocations";
     const MANAGE_USERS_LINK = "goToManageUsers";
 
-    const formLoaded = async () => {
+    const formRender = async () => {
       RouteHelper.AssociateHyperlinkClick(MANAGE_LOCATIONS_LINK, () =>
```

Comment:
Is this really an async function ? I can see that the implementation is non asynchronous.

Reply:
No, it is not. I will change it.

Reply:
Changed, good catch 😉